### PR TITLE
chore(release-calc): 0.3.0

### PR DESCRIPTION
**Pull request by TarikGul**
Release for `@substrate/calc` v0.3.0.

### CODE REVIEW

**Review by niklasad1 — APPROVED** [review]

File: calc/CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# Changelog
+
+All notable changes to this project will be documented in this file. See [standard-version](https://github.com/conventional-changelog/standard-version) for commit guidelines.
+
+## [0.3.0](https://github.com/paritytech/substrate-api-sidecar/tree/master/calc/) (2022-08-12)
+
+### BREAKING CHANGE
+
+- fix!(calc): fee calculation ([974](https://github.com/paritytech/substrate-api-sidecar/pull/974)) ([b4a8781](https://github.com/paritytech/substrate-api-sidecar/commit/b4a8781982b97ee6593290a5b137467bd0c93e22))
```
Comment (niklasad1) [inline calc/CHANGELOG.md:9]:
```suggestion
- fix(calc): fee calculation ([974](https://github.com/paritytech/substrate-api-sidecar/pull/974)) ([b4a8781](https://github.com/paritytech/substrate-api-sidecar/commit/b4a8781982b97ee6593290a5b137467bd0c93e22))
```

**Review by lexnv — APPROVED** [review]

**Review by jsdw — APPROVED** [review]

File: calc/CHANGELOG.md (hunk 1 above)
Comment (TarikGul) [reply calc/CHANGELOG.md:9]: So the bang is actually necessary via conventional commits for a breaking change, ie: https://www.conventionalcommits.org/en/v1.0.0/#commit-message-with--to-draw-attention-to-breaking-change

Comment (TarikGul) [reply calc/CHANGELOG.md:9]: But since i used it in the pushed commit, I think it might be best to keep it the same

Comment (niklasad1) [reply calc/CHANGELOG.md:9]: Ok, TIL

**Pull request merged**